src/Sylius/Behat/Element/Product/ShowPage/PricingElement.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Element\Product\ShowPage;
+
+use FriendsOfBehat\PageObjectExtension\Element\Element;
+
+final class PricingElement extends Element implements PricingElementInterface
+{
+    public function getPriceForChannel(string $channelName): string
+    {
+        /** @var NodeElement $priceForChannel */
+        $channelPriceRow = $this->getDocument()->find('css', sprintf('#pricing tr:contains("%s")', $channelName));
```

Comment:
Maybe we could move `sprintf('#pricing tr:contains("%s")', $channelName)` to getDefinedElements() function, like:

```
'priceForChannel' => '#pricing tr:contains("%channel%")',
```

and then:

```
$channelPriceRow = $this->getElement('priceForChannel', ['channel' => $channelName])
```

Reply:
I do not agree with you, in this case, `getDocument` take just one line of code, getElement make code less readable - "less is more"